lib/pbench/server/database/models/datasets.py

```diff
@@ -939,8 +942,8 @@ def create(**kwargs) -> "Metadata":
         for the referenced Dataset.
 
         NOTE: use this only for raw first-level Metadata keys, not dotted
-        paths like "dashboard.seen", which will fail key validation. Instead,
-        use the higher-level `set` helper.
+        paths like "global.seen", which will fail key validation. Instead,
```

Comment:
s/global.seen/global.dashboard.seen/ ?

or

s/global.seen/global.contact.name/ ?

Ditto line 1086.

Reply:
The reason why I called these out is that I get a certain dissonance from code comments which reflect something other than current practice.  I fully appreciate that everything after the first dot is just theoretical or for example, but there is or will be existing practice in which `seen` is a key which is not at the top level, and so it will make the documentation look "funny".  And since the [documentation](https://github.com/distributed-system-analysis/pbench/blob/ea1cc24f0ed2ac4c119702fb9ae22e846984b1d7/docs/API/metadata.md?plain=1#L42) explicitly points out `global.dashboard.seen`, it just seems like using `global.seen` elsewhere isn't the best idea.

But, I concede, in the abstract, it's perfectly fine.  So, I'm ready to move on.  🙂